Makefile

```diff
@@ -8,6 +8,7 @@ LOCAL_BOOTSTRAP_CONFIG = localenvoyconfig.yaml
 SECURE_LOCAL_BOOTSTRAP_CONFIG = securelocalenvoyconfig.yaml
 PHONY = gencerts
 ENVOY_IMAGE = docker.io/envoyproxy/envoy:v1.18.3
+GATEWAY_API_VERSION = $(grep "sigs.k8s.io/gateway-api" go.mod | awk '{print $2}')
```

Comment:
```suggestion
GATEWAY_API_VERSION = $(shell grep "sigs.k8s.io/gateway-api" go.mod | awk '{print $$2}')
```

Reply:
turns out we were pulling the latest `main` or whatever CRD yaml as well, we'll need to regenerated the CRDs as well before this will pass the check